modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js

```diff
@@ -144,3 +104,43 @@ export const BASELINE_CLAUSE_CONTRIBUTORS_CONFIGURATION = {
 		'com.liferay.portal.search.internal.spi.model.query.contributor.UserIdQueryPreFilterContributor',
 	],
 };
+
+export const CUSTOM_JSON_SXP_ELEMENT = {
+	description_i18n: {
+		en_US: Liferay.Language.get('editable-json-text-area'),
+	},
+	elementDefinition: {
+		category: 'custom',
+		configuration: {},
+		icon: 'custom-field',
+	},
+	title_i18n: {en_US: Liferay.Language.get('custom-json-element')},
+};
+
+export const DEFAULT_ADVANCED_CONFIGURATION = {};
+
+export const DEFAULT_BASELINE_SXP_ELEMENTS = [
+	boostAllKeywordsMatch,
+
+	// TODO Update once all elements are available
+
+	// TEXT_MATCH_OVER_MULTIPLE_FIELDS,
```

Comment:
@oliv-yu  We do have all the OOTB elements already available on master if this is what you meant here. :thinking:

Reply:
Or this this TODO for addressing the remaining work mentioned here? https://issues.liferay.com/browse/LPS-141101?focusedCommentId=2569876&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2569876

Reply:
It's that ticket, yes! We keep a copy of the baseline ootb elements on frontend for the "Add Blueprint" functionality.

Reply:
I see, thanks @oliv-yu ! :-)